lib/cfonb/operation.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 module CFONB
+  class Details; end
```

Comment:
No problem here, but if you still find this syntax ugly/weird you can also do this:
```suggestion
  Details = Class.new
```
It could suggest/remind that `Details` is a (dynamic) object, and that it could be amend later (on `OperationsDetails` registrations in our case).
But that's a matter of taste, so your call! 👍

Reply:
I agree with the suggestion, thanks a lot @RDeckard :)